src/main/java/papadom/commands/FindEventCommand.java

```diff
@@ -0,0 +1,25 @@
+package papadom.commands;
+
+import papadom.exceptions.IncorrectTaskInputFormatException;
+import papadom.utils.Parser;
```

Comment:
Should the import statement be at the bottom (above `import papadom.utils.ui` as it doesn't follow lexicographic ordering?